### ISSUE TITLE
support CSharp For Markup

### ISSUE THREAD

**Issue by sgf** [modern-forms/Modern.Forms#80]
https://github.com/VincentH-Net/CSharpForMarkup

Based on this feature, the UI definition can become modern and hierarchical.
In this way, we can abandon the imperative UI definition of winforms without introducing markup languages ​​(xml, xaml, etc.).

**Comment by jpobst** [comment]
I don't think this is something Modern.Forms will tackle.  One of the design goals is to provide a similar experience to Winforms, which is imperative UI.

However I imagine someone could write a NuGet package on top of Modern.Forms that would enable this experience.
